src/components/button/BasicButton.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <el-row>
-    <el-button>Default</el-button>
+    <el-button @click="onClick"> <slot></slot> </el-button>
```

Comment:
We should emit the same click event like so: `@click="$emit('click')"` so later we can use :
```
<basic-button @click="action">.....
```